<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Senso_vClass</class>
 <widget class="QMainWindow" name="Senso_vClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WinSenso</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Blei</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Rot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Gelb</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Grün</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Blau</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WinSenso V1.0  2017 FHWS FAK-ET Wolfgang Spieß</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWinSenso_V1_0">
    <property name="title">
     <string>V1.0</string>
    </property>
   </widget>
   <addaction name="menuWinSenso_V1_0"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="senso_v.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Senso_vClass</receiver>
   <slot>rotKlicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Senso_vClass</receiver>
   <slot>gelbKlicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Senso_vClass</receiver>
   <slot>blauKlicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Senso_vClass</receiver>
   <slot>runKlicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Senso_vClass</receiver>
   <slot>gruenKlicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>Senso_vClass</receiver>
   <slot>helpKlicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>windowShown()</signal>
  <slot>rotKlicked()</slot>
  <slot>gelbKlicked()</slot>
  <slot>ExitKlicked()</slot>
  <slot>gruenKlicked()</slot>
  <slot>blauKlicked()</slot>
  <slot>runKlicked()</slot>
  <slot>helpKlicked()</slot>
 </slots>
</ui>
